src/content/1.7/project/maintainers-guide/summary-github-lifecycle.md

```diff
@@ -56,6 +56,84 @@ We sometimes receive support requests on GitHub, ranging from questions about th
 
 There are some Issues which do not fit in the previous categories, such as [releases Issues](https://github.com/PrestaShop/PrestaShop/issues/20804). They serve a specific purpose.
 
+## Projects
+
+Available in the ‘[Projects](https://github.com/PrestaShop/PrestaShop/projects)’ tab of the PrestaShop’s GitHub repository, versions kanbans organize the Core team’s work. Both the product and the tech teams use it to prepare and plan the next developments and, since we deal with an open source project, the purpose of having those kanbans public is to help the community and the external mainteners to know what is at stake.
+
+### Definition of states
+
+A ticket can either be an issue or a pull request. A pull request should always be linked to the issue it fixes. 
+That's said, there should only have issues in the kanban and it always have an owner, be it a PM or a developer.
+
+Community contributions are no longer part of the kanban. Only tickets developed or fixed by the Core team are included in the kanban, as it should reflect the Core team’s work. For example, an issue reports a bug prioritized for the 1.7.8.0, the ticket must appear in the 1.7.8 kanban while the linked pull request must have the 1.7.8.0 milestone.
+
+Community contributions must be approved by the Core product team. Ideally before being reviewed, at least before being merged. Here is why:
+
+It is a bug? It must be reported in the functional specifications.
+It is an improvement? It must be validated, prioritized, and specified.
+
+A pull request that shows no activity for more than three months must be closed. If it is fixing a major bug or any topwatcher ticket, it will be back in the to do column again to be completed by the core team.
+
+Also, an important notion to keep in mind is the definition of ready. Several elements must be prepared upstream to get the ticket ready to be coded, otherwise it will not be included in the backlog.
+
+A ready ticket must contain an explicit title and description referring to:
+- validated & available specifications
+- validated & available mockups
+- validated & available wording
+- available acceptance tests
+
+#### Not ready
+
+**All the prioritized tickets to include in this version** but that need to be prepared by the core team - assigned to a PM or the lead dev as owner.
+
+_Who adds tickets to this column?_ Mostly the PM once they approved the ticket, but it could also be the developers.
+
+#### Backlog
+
+**All the ready tickets reviewed during the grooming** by the lead PM and the lead dev and discussed/assigned to a developer during the sprint planning.
```

Comment:
This is tricky because Grooming and Planning meetings are private. Better leave them out.
```suggestion
**All issues that are considered _ready_** by the lead PM and the lead dev. Issues in this column will be discussed to Core developers.
```

Reply:
Updated here: https://github.com/PrestaShop/docs/pull/807/commits/a24cd0d9c79059a6454232bbe0747d265d0af0d0